pkg/webview/view.proto

```diff
@@ -22,6 +22,15 @@ message BuildRecord {
 
   // The span id for this build record's logs in the main logstore.
   string span_id = 8;
+
+  enum BuildType {
```

Comment:
it is now!

Reply:
...wait, can you elaborate on why you want this to be top-level? Now that I'm making `UpdateType` and `TargetType` into their own enums, I'm running into some name collisions b/c they're both top-level.

Reply:
This is a common question! The style guide has an answer! :)

https://developers.google.com/protocol-buffers/docs/style#enums

> Prefer prefixing enum values instead of surrounding them in an enclosing message.

So it should be BUILD_TYPE_IMAGE

Reply:
🤔 yeah I saw that but once you generate the .pb file you get names like `BuildType_BUILD_TYPE_IMAGE`--but uh, if that's what the style guild says, I guess I'll go with it?

Reply:
ya, there's a long discussion on it here
https://github.com/golang/protobuf/issues/513
(and follow-up issues), the reasons are pretty complex and nuanced